addons/account_fleet/models/fleet_vehicle_log_services.py

```diff
@@ -0,0 +1,44 @@
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+class FleetVehicleLogServices(models.Model):
+    _inherit = 'fleet.vehicle.log.services'
+
+    account_move_line = fields.Many2one('account.move.line')
+    account_move_state = fields.Selection(related="account_move_line.parent_state", store=True)
+    account_move = fields.Many2one('account.move', related="account_move_line.move_id", store=True)
+    amount = fields.Monetary('Cost', compute="_compute_amount", inverse="_inverse_amount", readonly=False, store=True, tracking=True)
+    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', compute="_compute_vehicle_id", store=True, readonly=False, required=True)
+
+    @api.depends('account_move_line.vehicle_id')
+    def _compute_vehicle_id(self):
+        for service in self:
+            if service.account_move_line:
+                service.vehicle_id = service.account_move_line.vehicle_id
+
+    def _inverse_amount(self):
+        if any(service.account_move_line for service in self):
+            raise UserError(_("You cannot modify amount of services linked to an account move line. Do it on the related accounting entry instead."))
+
+    @api.depends('account_move_line.price_subtotal')
+    def _compute_amount(self):
+        for log_service in self:
+            if log_service.account_move_line:
+                log_service.amount = log_service.account_move_line.price_subtotal
+
+    def action_open_account_move(self):
+        self.ensure_one()
+        return {
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_model': 'account.move',
+            'target': 'current',
+            'name': _('Bills'),
```

Comment:
Where is the name visible? If it's needed, I guess it should be "Bill" since there's only one?